rust/kernel/prelude.rs

```diff
@@ -2,6 +2,9 @@
 
 //! The `kernel` prelude
 
+/// ICE if we use `--extern module`
+extern crate module;
```

Comment:
I need to retry this now that the minimum nightly was updated in the other PR. If it still crashes we should probably reduce it and file it.

Reply:
👍

Reply:
Still fails, also on stable. Filed: https://github.com/rust-lang/rust/issues/80946